restdocs-api-spec-openapi3-generator/src/main/kotlin/com/epages/restdocs/apispec/openapi3/OpenApi3Generator.kt

```diff
@@ -216,21 +216,23 @@ object OpenApi3Generator {
         val firstModelForPathAndMethod = modelsWithSamePathAndMethod.first()
         return Operation().apply {
             operationId = firstModelForPathAndMethod.operationId
-            summary = firstModelForPathAndMethod.summary
-            description = firstModelForPathAndMethod.description
+            summary = modelsWithSamePathAndMethod.map { it.summary }.find { !it.isNullOrBlank() }
+            description = modelsWithSamePathAndMethod.map { it.description }.find { !it.isNullOrBlank() }
             tags = modelsWithSamePathAndMethod.flatMap { it.tags }.distinct().nullIfEmpty()
             deprecated = if (modelsWithSamePathAndMethod.all { it.deprecated }) true else null
             parameters =
                     extractPathParameters(
-                        firstModelForPathAndMethod
+                        modelsWithSamePathAndMethod
```

Comment:
I think for the path parameters this changes is not needed. All items in `modelsWithSamePathAndMethod`  have the same path. This is asserted before. So it is save to just consider the first item in case of the path parameters